src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -98,8 +98,9 @@ setTimeout(async function () {
     let applyUpdateCapabilities;
     try {
       applyUpdateCapabilities = window.Blazor._internal.getApplyUpdateCapabilities();
-    } catch {
-      applyUpdateCapabilities = '';
+    } catch (error) {
+      console.warn(error);
+      applyUpdateCapabilities = '!' + error.stack;
```

Comment:
@MackinnonBuck maybe you're familiar with this code too?

Reply:
It's safe in the sense that this script is only used during development, and we don't risk exposing details about the application to the customer.

However, `stack` is a [non-standard-property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack), so it might not work on some browsers. What we could do is something like:

```js
const message = error.message || 'Error';
const stack = error.stack || '<no stack trace available>';
applyUpdateCapabilities = sendErrorToClient ? `!{message}: {stack}` : '';
```

So if no stack is available, we at least get the error message. And if the message is empty, we hopefully get a stack trace.

Reply:
Would this work?

```js
applyUpdateCapabilities = sendErrorToClient ? (error.stack || error.message || '<unknown error>') : '';
```

`error.stack` already includes the message.